evadb/executor/show_info_executor.py

```diff
@@ -51,15 +51,22 @@ def exec(self, *args, **kwargs):
             for db in databases:
                 show_entries.append(db.display_format())
         elif self.node.show_type is ShowType.CONFIG:
-            value = self.catalog().get_configuration_catalog_value(
-                key=self.node.show_val.upper(),
-            )
             show_entries = {}
-            if value is not None:
-                show_entries = {self.node.show_val: [value]}
+            # CONFIG is a special work, which is used to display all the configurations
+            if self.node.show_val.upper() == ShowType.CONFIG.name:
```

Comment:
Does this mean that `CONFIG` is a keyword we can not set?

Reply:
Yes, still confused between using `CONFIG` vs `CONFIGS` since we use show with plurals like `FUNCTIONS` or `DATABASES`

Reply:
We can have some list similar to `from evadb.catalog.sql_config import RESTRICTED_COL_NAMES` to prevent the access to the system key words.

Reply:
Should I add it in this list or create a new one?

Reply:
Let's have a new one.